### PR TITLE
Feature/output cassandra fulltext

### PR DESCRIPTION
**Pull request by gserranojc**
In this PR I must change the SDK, because the order of the fields in the schema and Dataframe is very important. 

If we create a table with primarykey((field1, field2),field3), the order in cassandra and the dataframe is field1, field2, field3, always behind the other fields of the table.

We have a bug in the ouput sdk to find schema in broadcast variable.

### CODE REVIEW

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala [hunk 1]
```diff
@@ -103,8 +125,11 @@ class CassandraOutput(keyName: String,
 }
 
 object CassandraOutput {
-  def getSparkConfiguration(configuration : Map[String, JSerializable]) : Seq[(String, String)] = {
-    val connectionHost = configuration.getString("connectionHost", "127.0.0.1")
+
+  final val DefaultHost = "127.0.0.1"
```
Comment (arinconstrio) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala]: As we discussed, default conf must be in a file

File: sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala [hunk 2]
```diff
@@ -108,37 +111,44 @@ abstract class Output(keyName: String,
     }
 
   protected def getTableSchemaFixedId(tbSchema: TableSchema): TableSchema = {
-    var tableName = tbSchema.tableName.split(Output.SEPARATOR)
-      .filter(name => !fixedBuckets.contains(name) && name != timeName).mkString(Output.SEPARATOR)
-    var fields = tbSchema.schema.fields.toSeq.filter(field =>
-      !fixedBuckets.contains(field.name) && field.name != timeName)
+    var tableName = tbSchema.tableName.split(Output.Separator)
```
Comment (arinconstrio) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala]: Pls mark this method to a refactor and open the related issue, I'm sure that we can remove this vars

File: sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala [hunk 3]
```diff
@@ -172,8 +182,11 @@ abstract class Output(keyName: String,
 
 object Output {
 
-  final val SEPARATOR = "_"
-  final val ID = "id"
+  final val Separator = "_"
+  final val Id = "id"
+  final val FixedAggregation = "fixedAggregation"
```
Comment (arinconstrio) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala]: Aggregation and multiplexer maybe aren't related to the output, I think that we need to move this values to other class.

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala (hunk 1 above)
Comment (gserranojc) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala]: We have a issue for move all default configurations values.

File: sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala (hunk 2 above)
Comment (gserranojc) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala]: Ok! I think that is possible, I add one TODO and then generate a new issue.

File: sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala (hunk 3 above)
Comment (gserranojc) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala]: We need change all multiplexer operations to rollup level as we discuss the last week.
This three constants, where do you think that they must be declared?

Comment (arinconstrio) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala]: My doubt is about FixedAggregation, are these aggregation fields global? or related to the output?

**Comment by arinconstrio** [comment]
We can’t automatically merge this pull request. pls merge this pr with master

File: sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala (hunk 3 above)
Comment (gserranojc) [inline sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala]: Now is related to ouput configuration.

**Pull request merged**